### ISSUE TITLE
Outdated documentation and examples

### ISSUE THREAD

**Issue by MaciekWin3** [dotnet-websharper/core#1356]
I've recently started tinkering with WebSharper, and it's been mostly great. However, after trying to do some more advanced things and figuring them out by myself, I've found it hard to locate relevant information in the documentation. The documentation and examples seem out of date, stuck at version 4 while there's active development on version 7. I'm pretty sure WebSharper is an interesting project, but the outdated documentation and examples are a turn-off. Any plans to update them with the upcoming release? I understand that open-source projects can be tough to maintain, and I respect the effort behind WebSharper. I'm ready to pitch in if you need help with the docs or if you can point me in the right direction. Better docs would make life easier for newcomers like me. Looking forward to your response and any advice you can offer.

**Comment by granicz** [comment]
You are absolutely right: the documentation does smell way too old and outdated (yet, it's mostly NOT outdated, don't let the URL with 4.x mislead you!) and the places for relevant examples are not readily apparent to newcomers. One thing is certain: there are TONS of features not documented outside of tickets, mostly those added since WebSharper 4. Volume-wise, there is massive amounts of text scattered across multiple places/repos, and there are hundreds of examples/repos/snippets that need updating. Your help is greatly appreciated - please find me on X/LinkedIn/Slack/Discord to discuss, or feel free to open up a discussion here on GitHub.

Some areas where contribution would yield the most benefit IMHO (in no particular order):

0) Beginner-friendly tutorials and blog articles for WebSharper 7 -> these are absolutely KEY and we can help spreading the word
1) Reworking the documentation -> the [new repo](https://github.com/dotnet-websharper/docs) contains some material brought over from the old one, but mostly missing the rest as of now, deploys to https://docs.websharper.com -> not yet linked on the [WebSharper site](https://websharper.com) but should be.
2) Documenting all new WS4+ features into the new docs repo -> WebSharper 7 is easily the most effective tool for full-stack .NET web development, and not having full documentation is a killer. Some of the things missing are in UI templating (`ws-anchor`, `ws-dom`, `InputType.*`, etc.), full-stack features (hydration, server-side event bindings, etc.), remoting enhancements and Fable.Remoting compatibility, support for web components and their templates, deep integration with React/JSX, a multitude of new bindings, etc. We recently started to mark some of these tickets with the "to-be-documented" label, here are those in the core repo: https://github.com/dotnet-websharper/core/labels/to-be-documented, and do check the other repos as well.
3) Beefing up [Try F# on WASM](https://github.com/fsbolero/TryFSharpOnWasm) to work with WebSharper code as well -> this could replace the current [Try WebSharper](https://try.websharper.com) site.
4) Lots of tweets, toots, social media activity -> need community champions and leaders!
5) Streamlining CI to auto-deploy test/demo projects to GitHub Pages from each binding repo -> this creates more live examples.
6) Organizing online/in-person events for WebSharper -> we can offer help/sponsorship
7) Posting WebSharper jobs -> we can feature these on the website and give more visibility to them in social media, etc.
8) Creating new WebSharper bindings -> we can help with build script templates, etc.
9) Publishing WebSharper demo applications, project templates, case studies, etc -> would love to see what others are doing with WebSharper!
10) Updating/fixing existing sample repos (some are in https://github.com/websharper-samples), Try WebSharper snippets
11) Collecting up WebSharper resources for newcomers, adding to @Tarmil 's https://github.com/Tarmil/awesome-websharper, etc.
...
